build.gradle.kts

```diff
@@ -12,6 +12,7 @@ buildscript {
         classpath("com.deploygate:gradle:2.0.1")
         classpath("com.dicedmelon.gradle:jacoco-android:0.1.4")
         classpath("org.jlleitschuh.gradle:ktlint-gradle:8.2.0")
+        classpath("com.google.gms:google-services:4.2.0")
```

Comment:
Android Studio 3.4.1の場合

`com.google.gms:google-services:4.2.0` を使用するつもりが、Android Studioで新しい更新プログラムがあることが通知されたので、最新版の `com.google.gms:google-services:4.3.0` に更新し、プロジェクトをコンパイルしたら警告が表示される様になったので、バージョンを `4.2.0` に戻した。

```
WARNING: API 'variant.getMergeResources()' is obsolete and has been replaced with 'variant.getMergeResourcesProvider()'.
It will be removed at the end of 2019.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getMergeResources(), use -Pandroid.debug.obsoleteApi=true on the command line to display more information.
Affected Modules: app
```

参照: https://github.com/google/play-services-plugins/issues/65